src/sentry/static/sentry/app/components/pill.jsx

```diff
@@ -29,19 +29,80 @@ class Pill extends React.Component {
   };
 
   render() {
-    const {name, children, className, ...props} = this.props;
-    const [extraClass, renderedValue] = this.renderValue();
+    const {name, children, ...props} = this.props;
+    const [renderedValue] = this.renderValue();
 
     return (
-      <li className={classNames(className, extraClass)} {...props}>
-        <span className="key">{name}</span>
-        <span className="value">
+      <StyledPill {...props}>
+        <PillName>{name}</PillName>
+        <PillValue>
           {renderedValue}
           {children}
-        </span>
-      </li>
+        </PillValue>
+      </StyledPill>
     );
   }
 }
 
+const StyledPill = styled('li')`
+  white-space: nowrap;
+  margin: 0 10px 10px 0;
+  display: flex;
+  border: 1px solid #d0c9d7;
+  border-radius: 3px;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.04);
+  line-height: 1.2;
+  max-width: 100%;
+  &.true .value {
+    background: #493e54;
+    border: 1px solid #c7dbbd;
+    margin: -1px;
+    color: #6a726c;
+  }
+  &.false .value {
+    background: #fff9f9;
+    border: 1px solid #e5c4c4;
+    margin: -1px;
+    color: #766a6a;
+  }
+  &:last-child {
+    margin-right: 0;
+  }
+`;
+
+const SharedPillSpanStyles = css`
+  padding: ${space(0.5)} ${space(1)};
+  min-width: 0;
+  white-space: nowrap;
+`;
+
+const PillName = styled('span')`
+  ${SharedPillSpanStyles}
+`;
+
+const PillValue = styled('span')`
+  ${SharedPillSpanStyles}
+  background: #fbfbfc;
+  border-left: 1px solid #d8d2de;
+  border-radius: 0 3px 3px 0;
+  font-family: Monaco, monospace;
+  max-width: 100%;
+  text-overflow: ellipsis;
+  > a {
+    max-width: 100%;
+    display: inline-block;
+    vertical-align: text-bottom;
+  }
+  .pill-icon,
```

Comment:
We should check to see if this is used still

Reply:
We need to keep the `display: inline-block` for the `text-overflow` and the removign the `vertical-align` impacts the UI .